packages/dev/loaders/src/SPLAT/splatFileLoader.ts

```diff
@@ -74,10 +77,24 @@ export class SPLATFileLoader implements ISceneLoaderPluginAsync, ISceneLoaderPlu
         ".ply": { isBinary: true },
     } as const satisfies ISceneLoaderPluginExtensions;
 
+    /**
+     * Shall data stay in ram for editing/cloning
+     */
+    public static KEEP_IN_RAM = false;

```

Comment:
I guess the idea here is to allow users to set the default for this if it is not provided as an option. One issue with making it a static member of the class is that when we switch to having loaders by loaded dynamically, then there will be no way to set this default option value without statically loading the whole loader. Maybe another way to handle this would be to put both the defaults and the options in the splatLoadingOptions.ts file. Maybe something like:

```ts
export const defaultSPLATLoadingOptions = {
  keepInRam: false,
};

export type SPLATLoadingOptions = typeof defaultSPLATLoadingOptions;
```

Then we are saying users are free to modify the default options and only need to import the `splatLoadingOptions` module to do so.